lazyflow/operators/ioOperators/ioOperators.py

```diff
@@ -559,51 +561,54 @@ def setupOutputs(self):
         self.outputs["WriteImage"].meta.shape = (1,)
         self.outputs["WriteImage"].meta.dtype = object
 
-        self.f = self.inputs["hdf5File"].value
-        hdf5Path = self.inputs["hdf5Path"].value
+        self.f = self.inputs["h5N5File"].value
+        h5N5Path = self.inputs["h5N5Path"].value
 
         # On windows, there may be backslashes.
-        hdf5Path = hdf5Path.replace('\\', '/')
+        h5N5Path = h5N5Path.replace('\\', '/')
 
-        hdf5GroupName, datasetName = os.path.split(hdf5Path)
-        if hdf5GroupName == "":
+        h5N5GroupName, datasetName = os.path.split(h5N5Path)
+        if h5N5GroupName == "":
             g = self.f
         else:
-            if hdf5GroupName in self.f:
-                g = self.f[hdf5GroupName]
+            if h5N5GroupName in self.f:
+                g = self.f[h5N5GroupName]
             else:
-                g = self.f.create_group(hdf5GroupName)
+                g = self.f.create_group(h5N5GroupName)
 
         dataShape=self.Image.meta.shape
         self.logger.info( "Data shape: {}".format(dataShape))
 
         dtype = self.Image.meta.dtype
         if isinstance(dtype, numpy.dtype):
             # Make sure we're dealing with a type (e.g. numpy.float64),
-            #  not a numpy.dtype
+            # not a numpy.dtype
             dtype = dtype.type
         # Set up our chunk shape: Aim for a cube that's roughly 512k in size
         dtypeBytes = dtype().nbytes
 
         tagged_maxshape = self.Image.meta.getTaggedShape()
         if 't' in tagged_maxshape:
             # Assume that chunks should not span multiple t-slices,
-            #  and channels are often handled separately, too.
+            # and channels are often handled separately, too.
             tagged_maxshape['t'] = 1
 
         if 'c' in tagged_maxshape:
             tagged_maxshape['c'] = 1
 
-        self.chunkShape = determineBlockShape( list(tagged_maxshape.values()), 512000.0 / dtypeBytes )
+        self.chunkShape = determineBlockShape(list(tagged_maxshape.values()), 512000.0 / dtypeBytes)
 
         if datasetName in list(g.keys()):
             del g[datasetName]
-        kwargs = { 'shape' : dataShape, 'dtype' : dtype,
-            'chunks' : self.chunkShape }
+        kwargs = {'shape': dataShape, 'dtype': dtype,
+                  'chunks': self.chunkShape}
         if self.CompressionEnabled.value:
```

Comment:
So things at z5 are moving fast. Turns out they do have compression enabled per default because they target really large data. In any case, with the two menu entries we have, the uncompressed option should really be uncompressed :)
so this `if`, needs and `else`, or just define some default `kwargs` without compression before the if clause